File: pgm/ABC_Java/ABC_452_A.java
import java.io.*;
// import java.math.*;
// import java.time.*;
import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        reader.close();
        final String Yes = "Yes";
        final String No = "No";
        final int M1 = 1;
        final int M3 = 3;
        final int M5 = 5;
        final int M7 = 7;
        final int M9 = 9;
        final int D3 = 3;
        final int D5 = 5;
        final int D7 = 7;
        final int D9 = 9;
        final int one = 1;
        final int zero = 0;
        final int M = Integer.parseInt (arr[zero]);
        final int D = Integer.parseInt (arr[one]);
        Set<Gosekku> g = new HashSet<>();
        g.add (new Gosekku (M1, D7));
        g.add (new Gosekku (M3, D3));
        g.add (new Gosekku (M5, D5));
        g.add (new Gosekku (M7, D7));
        g.add (new Gosekku (M9, D9));
        System.out.println (g.contains (new Gosekku (M, D)) ? Yes : No);
        System.exit (0);
    }
}

class Gosekku {
    private final int M, D;
    public Gosekku (int M, int D) {
        this.M = M;
        this.D = D;
    }

    @Override
    public boolean equals (Object o) {
        if (this == o)
            return true;
        if (!(o instanceof Gosekku g))
            return false;
        return M == g.M && D == g.D;
    }

    @Override
    public int hashCode() {
        return Objects.hash (M, D);
    }
}

File: pgm/ABC_C#/ABC_452_A.cs
using System;
using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
// using System.Linq;
// using System.Text;
// using System.Text.RegularExpressions;

class Program
{
    static void Main()
    {
        string[] arr = Console.ReadLine().Split(' ');
        const string Yes = "Yes";
        const string No = "No";
        const int M1 = 1;
        const int M3 = 3;
        const int M5 = 5;
        const int M7 = 7;
        const int M9 = 9;
        const int D3 = 3;
        const int D5 = 5;
        const int D7 = 7;
        const int D9 = 9;
        const int one = 1;
        const int zero = 0;
        int M = int.Parse(arr[zero]);
        int D = int.Parse(arr[one]);
        HashSet<Gosekku> g = new HashSet<Gosekku>();
        g.Add(new Gosekku(M1, D7));
        g.Add(new Gosekku(M3, D3));
        g.Add(new Gosekku(M5, D5));
        g.Add(new Gosekku(M7, D7));
        g.Add(new Gosekku(M9, D9));
        Console.WriteLine(g.Contains(new Gosekku(M, D)) ? Yes : No);
        Environment.Exit(0);
    }
    readonly record struct Gosekku(int M, int D);
}